src/app/space/create/deployments/apps/delete-deployment-modal.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, ElementRef, EventEmitter, Input, OnInit, Output, ViewChild, ViewEncapsulation } from '@angular/core';
+
+import { NotificationType } from 'ngx-base';
+import { ModalDirective } from 'ngx-bootstrap/modal';
+import { Subscription } from 'rxjs';
+import { Environment } from '../models/environment';
+
+import { DeploymentsService } from '../services/deployments.service';
+import { DeploymentCardComponent } from './deployment-card.component';
+
+@Component({
+  host: {
+    class: 'create-dialog'
+  },
+  encapsulation: ViewEncapsulation.None,
+  selector: 'delete-deployment-modal',
+  templateUrl: './delete-deployment-modal.component.html'
+})
+export class DeleteDeploymentModal {
+
+  @Input() host: ModalDirective;
+  @Input() applicationId: string = '';
+  @Input() environmentName: string = '';
+  @Output() deleteEvent = new EventEmitter();
+
+  constructor(
+    private deploymentsService: DeploymentsService) {
```

Comment:
Sorry, missed this on the last round. Now that we're emitting events rather than directly issuing the deletion call from the modal we don't need this service injected anymore. There are also several imports to be removed.

Reply:
Good catch